Makefile

```diff
@@ -12,6 +12,8 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+DOCKER_TAG=latest
```

Comment:
we are tagging image based on which branch we are on

Reply:
You are tag_for_branch() function need tag pararms, but you did't pass it